packages/devtools_app/lib/src/screens/memory/memory_analyzer.dart

```diff
@@ -331,36 +338,39 @@ Map<String, List<String>> drillIn(
   final matcher = ObjectMatcher((className, fields, value) {
     final key = '$className.${fields.join(".")}';
     result.putIfAbsent(key, () => []);
-    result[key].add(value);
+    result[key]!.add(value);
   });
 
-  for (final ClassReference classRef in references) {
-    if (!matcher.isClassMatched(classRef.name)) continue;
+  for (final ClassReference classRef
+      in references as Iterable<ClassReference>) {
+    if (classRef.name == null || !matcher.isClassMatched(classRef.name))
```

Comment:
nit style: use brackets {} if the continue or return statement breaks to a second line

Reply:
done